ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -311,11 +360,17 @@ private void readWithProgress(
       int requestId,
       ResponseBody responseBody) throws IOException {
     Reader reader = responseBody.charStream();
+    ProgressResponseBody progressResponseBody = (ProgressResponseBody)responseBody;
     try {
       char[] buffer = new char[MAX_CHUNK_SIZE_BETWEEN_FLUSHES];
       int read;
       while ((read = reader.read(buffer)) != -1) {
-        ResponseUtil.onDataReceived(eventEmitter, requestId, new String(buffer, 0, read));
+        ResponseUtil.onIncrementalDataReceived(
```

Comment:
Yep, this changes the event names because events change meaning.

Which clients of RCTNetworking are there other than XMLHttpRequest?